diracx/templates/diracx/deployment.yaml

```diff
@@ -147,6 +147,12 @@ spec:
             {{- end }}
           {{- end }}
           volumeMounts: {{ toYaml $commonVolumeMounts | nindent 12 }}
+          env:
+            # Required by FastAPIInstrumentor
+            # to follow semantic conventions for HTTP metrics
+            # https://opentelemetry.io/docs/specs/semconv/http/http-metrics/
+            - name: OTEL_SEMCONV_STABILITY_OPT_IN
```

Comment:
I think this is such a low level that you can probably expot it directly in `otel.py`

Reply:
Wait, I can actually add it in the calling module